api/docker/caddy/Caddyfile

```diff
@@ -2,49 +2,71 @@
     # Debug
     {$DEBUG}
 
-    http_port 3001
-    https_port 3443
     auto_https disable_redirects
 }
 
-{$SERVER_NAME}
+:3000 {
+	log {
+	    level DEBUG
+	}
 
-log
+	handle_path /api* {
+	    header X-Forwarded-Prefix /api
```

Comment:
This is the magic, with this header the api can handle it perfectly that it's behind a subpath of a reverse proxy.

Reply:
Leave comment, this could be a small performance drop

Reply:
done